src/controllers/campaign/stepTwo.ts

```diff
@@ -0,0 +1,40 @@
+import { AppResponse } from '@/common/utils';
+import AppError from '@/common/utils/appError';
+import { catchAsync } from '@/middlewares';
+import { campaignModel as campaign } from '@/models/campaignModel';
+import { Request, Response } from 'express';
+
+const secondStep = catchAsync(async (req: Request, res: Response) => {
+	const { user } = req;
+	const { title, fundraiser, goal } = req.body;
+	const { campaignId } = req.params;
+
+	if (!user) {
+		throw new AppError('User not found!. Please login');
+	}
+
+	if (!title || !fundraiser || !goal) {
+		throw new AppError('Please provide the necessary credentials');
+	}
+
+	const currentDateDeadline = Date.now();
+
+	const step2Campaign = await campaign.findByIdAndUpdate(
+		campaignId,
+		{
+			title,
+			fundraiser,
+			goal,
+			deadline: currentDateDeadline,
```

Comment:
using the Date.now() means the campaign will expire right now?? why is this so

Reply:
the Date.now(),  was just added as an extra measure to make sure the campaign was created at that particular time. Removing it if not needed